src/Timezone/TimezoneAwareInterface.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of the Sonata Project package.
+ *
+ * (c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sonata\IntlBundle\Timezone;
+
+/**
+ * @author Javier Spagnoletti <phansys@gmail.com>
+ */
+interface TimezoneAwareInterface
```

Comment:
Do we need the interface suffix here? Sounds a little tautologic to me.

Reply:
It's part of symfony standard https://symfony.com/doc/current/contributing/code/standards.html#naming-conventions

Reply:
One more reason not to follow this standard IMO

Reply:
All (or almost all) interface of SonataAdminBundle have the interface suffix.

Personally I do like this standard, it sometimes avoid a conflict between the name of the interface and the name of the implementation. Like
![image](https://user-images.githubusercontent.com/9052536/83347350-b9d33c00-a324-11ea-9b4e-abd06b38fc78.png)

Reply:
> All (or almost all) interface of SonataAdminBundle have the interface suffix.

Then we should probably keep that suffix. But I wouldn't be proud of that, having to resolve the conflict you pointed out above is often a code smell. And sure enough, it looks like `Datagrid` was implemented with `symfony/form`, so it should probably be named `SymfonyFormDatagrid`… but wasn't why is that? Well it's because [the `DatagridInterface` itself relies on `symfony/form`](https://github.com/sonata-project/SonataDatagridBundle/blob/16877cd7586b7e7f0938ba27ed4a618a364967da/src/Datagrid/DatagridInterface.php#L44), with is a clear dependency inversion principle violation.

That standard does not avoid a conflict so much as avoid thinking about what's wrong with the architecture IMO.

Reply:
I'm used to the Symfony's naming convention, but I agree about the DIP analysis.
I suggest to respect the adopted naming convention here and decide how to proceed about these cases in a more general way in a separate issue.
WDYT?

Reply:
In the TimezoneAware case, if we remove the suffix, we'll have a conflict between the Trait and the Interface.

I'm also used to the Symfony's naming convention, for consistency we should keep this suffix but I see no shame in this one. When I require a file, it's easier to know if I require a Class, a Trait, an Interface, ...

DIP violation is something we can solve without removing the suffix.